backend/data/processors/conditions_preprocessing.py

```diff
@@ -432,6 +442,134 @@ def map_word_to_program_type(processed, regex_word, type):
         flags=re.IGNORECASE,
     )  # hard to capture a generic case?
 
+#adds handbook notes and removes phrasing
```

Comment:
Another small style nitpick but, ideally this comment would be inside the function as a docstring.
It may also be worth including an example in the docstring as though I'm able to tell what this does based off the details of the pull request, I don't know if it would be immediately obvious to someone looking at this function after the fact.